app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -306,4 +290,25 @@ def save_storage_files_inventory(storage, hashes)
     deletes = storage.storage_files(true).dup
     save_inventory_multi(:storage_files, storage, hashes, deletes, [:name])
   end
+
+  def find_host(h, ems_id)
+    if h[:ems_ref]
+      _log.debug "EMS ID: #{ems_id} Host database lookup - ems_ref: [#{h[:ems_ref]}] ems_id: [#{ems_id}]"
+      found = Host.find_by_ems_ref_and_ems_id(h[:ems_ref], ems_id)
+    end
+
+    if found.nil?
```

Comment:
@bdunne doesn't this local variable need to be set first in case the code inside the `if` above does not run